modules/persistent-storage-csi-gcp-filestore-wif.adoc

```diff
@@ -0,0 +1,126 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_csi-google_cloud_file.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-gcp-filestore-wif_{context}"]
+= Preparing to install the {gcp-short} Filestore CSI Driver Operator with Workload Identity
+
+If you are planning to use {gcp-wid-short} with Google Compute Platform Filestore, you must obtain certain parameters that you will use during the installation of the {gcp-short} Filestore Container Storage Interface (CSI) Driver Operator.
+
+.Prerequisites
+* Access to the cluster as a user with the cluster-admin role.
+
+// Put note in install area of docs to remind users to take note of the identity pool ID and the provider ID
+
+.Procedure
+
+To prepare to install the {gcp-short} Filestore CSI Driver Operator with Workload Identity:
+
+. Obtain the project number:
+
+.. Obtain the project ID by running the following command:
++
+[source, terminal]
+----
+$ export PROJECT_ID=$(oc get infrastructure/cluster -o jsonpath='{.status.platformStatus.gcp.projectID}')
+----
+
+.. Obtain the project number, using the project ID, by running the following command:
++
+[source, terminal]
+----
+$ gcloud projects describe $PROJECT_ID --format="value(projectNumber)"
+----
+
+. Find the identity pool ID and the provider ID:
++
+During cluster installation, the names of these resources is provided to `ccoctl` with the `--name parameter`. See Step 3 of Section _Creating {gcp-short} resources with the Cloud Credential Operator utility_.
```

Comment:
```suggestion
During cluster installation, the names of these resources are provided to the Cloud Credential Operator utility (`ccoctl`) with the `--name parameter`. See "Creating {gcp-short} resources with the Cloud Credential Operator utility".
```
(would avoid using the actual number as it can change later and make this inaccurate)

Reply:
Fixed.